cloud/blockstore/tests/loadtest/local-overflow/local-tablet-version-1.txt

```diff
@@ -1,8 +1,14 @@
 Vertices {
     Test {
+        Name: "create_and_shoot_volume"
         CreateVolumeRequest {
+            DiskId: "@volume"
             BlocksCount: 20000000
             BlockSize: 4096
+            PerformanceProfile {
+                MaxReadIops: 1
+                ThrottlingEnabled: 0
```

Comment:
Disable throttling otherwise test might not encounter Reassign event (and canonization check fails) because "Throttled" is also E_REJECTED 

https://github.com/ydb-platform/nbs/blob/f9a8864ee13ba900f61b89eb9c976e6074bf0bff/cloud/blockstore/libs/storage/volume/volume_actor_throttling.cpp#L169